shared-helpers/src/views/accounts/CustomIcons.tsx

```diff
@@ -60,3 +60,22 @@ export const Profile = (
     />
   </svg>
 )
+
```

Comment:
We shouldn't be adding icons that existing the public libraries we already use, since we're not on hero icons yet we can use the existing font-awesome library, will look like `import { faGear } from "@fortawesome/free-solid-svg-icons"`